pipelines/_ingredients/README.md

```diff
@@ -1,7 +1,8 @@
 # Pipeline Definition Ingredients
 
 This directory contains ingredient pipeline definition YAML files.
+
 The pipelines defined here are intended to be used as building blocks for more complex pipelines.
-As such, they are not intended to be used directly by end-users for science purposes.
+As such, they are not intended to be used directly by end-users.
```

Comment:
I do like ending the sentence with "for science purposes". What was the rationale to remove it?

Reply:
I agree with you after our chat just now - adding this back in for the three files impacted here.